src/lib/features/instance-stats/getProductionChanges.ts

```diff
@@ -0,0 +1,44 @@
+import { type Db } from 'lib/server-impl';
+import { GetActiveUsers } from './getActiveUsers';
+
+export type GetProductionChanges = () => Promise<{
+    last30: number;
+    last60: number;
+    last90: number;
+}>;
+
+export const createGetProductionChanges =
+    (db: Db): GetProductionChanges =>
+    async () => {
+        const productionChanges = await db
+            .select({
+                last_month: db.raw(
+                    "SUM(CASE WHEN day > NOW() - INTERVAL '1 month' THEN updates END)",
```

Comment:
Should we change this to 30 days as well, to make it consistent with the other two cases?